physical/gcs/gcs.go

```diff
@@ -140,30 +158,24 @@ func NewBackend(c map[string]string, logger log.Logger) (physical.Backend, error
 		"ha_enabled", haEnabled,
 		"max_parallel", maxParallel,
 	)
-	logger.Debug("creating client")
-
-	// Client
-	opts := []option.ClientOption{option.WithUserAgent(useragent.String())}
-	if credentialsFile := c["credentials_file"]; credentialsFile != "" {
```

Comment:
This has been deprecated for a very, very long time.

Reply:
@kalafut this might be fine to remove since we've had this notice for ~2yrs, but we should call this out in the changelog.